File: python/412_fizz_buzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        res = []
        
        for i in range(1, n+1):
            if i % 3 == 0:
                tmp = "Fizz"
                if i % 5 == 0:
                    tmp += "Buzz"
            elif i % 5 == 0:
                tmp = "Buzz"
            else:
                tmp = str(i)
            
            res.append(tmp)
        
        return res
        
                

File: swift/412_fizz_buzz.swift
class Solution {
  func fizzBuzz(_ n: Int) -> [String] {
    var res: [String] = []

    for i in 1...n {
      var ans: String = ""

      if i % 3 == 0 {
        ans = "Fizz"
        if i % 5 == 0 {
          ans += "Buzz"
        }
      } else if i % 5 == 0 {
        ans = "Buzz"
      } else {
        ans = String(i)
      }

      res.append(ans)
    }

    return res
  }
}
